app/Http/Controllers/CampController.php

```diff
@@ -270,8 +270,11 @@ public function getCampRecord(Request $request, Validate $validate)
             $livecamp = Camp::getLiveCamp($filter);
             if ($livecamp) {
                 $livecamp->nick_name=isset($livecamp->nickname->nick_name) ? $livecamp->nickname->nick_name : "No nickname associated";
```

Comment:
Replace it with `$livecamp->nick_name= $livecamp->nickname->nick_name ?? "No nickname associated";`
And "No nickname associated" should be in the messages.php file